thriftbp/prometheus.go

```diff
@@ -14,22 +14,23 @@ const (
 	exceptionLabel           = "thrift_exception_type"
 	baseplateStatusLabel     = "thrift_baseplate_status"
 	baseplateStatusCodeLabel = "thrift_baseplate_status_code"
+	slugLabel                = "thrift_slug"
```

Comment:
similar to kyle's comment above, here it's really hard to tell the difference between `slugLabel` and `serviceLabel`.